rule_packages/c/Contracts5.json

```diff
@@ -0,0 +1,38 @@
+{
+  "CERT-C": {
+    "ERR32-C": {
+      "properties": {
+        "obligation": "rule"
+      },
+      "queries": [
+        {
+          "description": "Do not rely on indeterminate values of errno.",
+          "kind": "problem",
+          "name": "Do not rely on indeterminate values of errno",
+          "precision": "high",
+          "severity": "error",
+          "short_name": "DoNotRelyOnIndeterminateValuesOfErrno",
+          "tags": ["correctness"]
+        }
+      ],
+      "title": "Do not rely on indeterminate values of errno"
+    },
+    "ERR33-C": {
+      "properties": {
+        "obligation": "rule"
+      },
+      "queries": [
+        {
+          "description": "Detect and handle standard library errors.",
```

Comment:
I would consider moving the "Undetected failures can lead to unexpected or undefined behavior" part of the alert msg to here instead